Mathlib/SetTheory/Cardinal/Aleph.lean

```diff
@@ -108,7 +108,7 @@ def omega' : Ordinal.{u} ↪o Ordinal.{u} where
 theorem coe_omega' : omega' = enumOrd {x | IsInitial x} :=
   rfl
 
-theorem strictMono_omega' : StrictMono omega' :=
+theorem omega_strictMono' : StrictMono omega' :=
```

Comment:
Yes, the prime is misplaced. Also, `strictMono` is a special case in that it suffixes names instead of prefixing them, I learned that the hard way in #17843.